memory/combat_manager.py

```diff
@@ -207,12 +205,8 @@ def read_next_combat_enemy(self: Self) -> None:
                     self.next_combat_enemy = None
                     return
                 spell_power = self.memory.read_float(combat_move_ptr + 0x30)
-                guid_ptr = self.memory.follow_pointer(
-                    ongoing_move_ptr, [0xF8, 0xF0, 0x18, 0x0]
-                )
-                guid_ptr = self.memory.follow_pointer(
-                    ongoing_move_ptr, [0xF8, 0xF0, 0x18, 0x0]
-                )
+                guid_ptr = self.memory.follow_pointer(ongoing_move_ptr, [0xF8, 0xF0, 0x18, 0x0])
+                guid_ptr = self.memory.follow_pointer(ongoing_move_ptr, [0xF8, 0xF0, 0x18, 0x0])
```

Comment:
There seems to be a duplicate line here (both before and after format). @Eein can you confirm if this should be removed?